redirector/common.py

```diff
@@ -80,13 +80,26 @@ def get_ddb_table():
     return boto3.resource("dynamodb").Table(MANIFEST_TABLE_NAME)
 
 
+def get_file_pathname(file):
+    # want to start the path after "All Files/<BoxFolderName>/"
+    file_path_collection = file.path_collection
+    file_path_names = [fp.get().name for fp in file_path_collection["entries"][2:]]
```

Comment:
Does this `2` index work generally, or only when the folder is a direct child of "All Files" in Box?  We might want to specify this relative to the index of common.BOX_FOLDER_ID, i.e.,

```python
start_index = [e.id for e in file_path_collection["entries"]].index(common.BOX_FOLDER_ID) + 1
file_path_names = [fp.get().name for fp in file_path_collection["entries"][start_index:]
```

Reply:
This is a better idea and I implemented it. I wasn't comfortable with the "2" being hardcoded either.